server/db/schema/01_users.sql

```diff
@@ -0,0 +1,43 @@
+DROP TABLE IF EXISTS parents CASCADE;
+DROP TABLE IF EXISTS baby CASCADE;
+DROP TABLE IF EXISTS parent_baby CASCADE;
+DROP TABLE IF EXISTS LOGS CASCADE;
+
+
+CREATE TABLE parents (
```

Comment:
@mhakbar rename the `parents` to `parent`.

Reply:
@mhakbar also add a new column for password. I forgot to add it to the erd but the parent's email and password will be how the user will login.